src/components/home.js

```diff
@@ -32,12 +33,46 @@ function Home(props) {
           setLastUpdated(response.data.statewise[0].lastupdatedtime.slice(0, 15)+response.data.statewise[0].lastupdatedtime.slice(18));
           setDeltas(response.data.key_values[0]);
           setFetched(true);
+          getSummaryData(response.data);
         })
         .catch((err)=>{
           console.log(err);
         });
   };
 
+  const getSummaryData = (data) => {
+    // This should be moved to utils
+    let confirmed = 0;
+    let recoveries = 0;
+    let deaths = 0;
+    data.statewise.map((state, index) => {
+      if (index !== 0) {
+        confirmed += parseInt(state.confirmed);
+        recoveries += parseInt(state.recovered);
+        deaths += parseInt(state.deaths);
+      }
+    });
+
+    const todayDate = data.key_values[0].lastupdatedtime.split(',')[0];
+    const todayData = data.cases_time_series.filter((x) => x.date.trim() === todayDate)[0];
+    const dailyConfirmed = todayData ? todayData.dailyconfirmed: '0';
+
+    const dateTimeString = new Date().toLocaleString('en-IN', {timeZone: 'Asia/Kolkata'}).slice(0, -3);
+    const twitterString = `COVID-19 India : 📊 as of ${dateTimeString} IST
+    Total Confirmed : ${confirmed}
+    Total Recovered : ${recoveries}
+    Total Deceased  : ${deaths}
+
+    Number of cases reported today: ${dailyConfirmed}
+
+    Follow @covid19indiaorg
+
+    #COVID19India #SocialDistancing
+    More @`;
+
+    setSummaryData(twitterString);
+  };
```

Comment:
The string would show as of today, there are so and so confirmed, recovered, deaths. But the cases reported today would be the number of cases would be the number of confirmed would be as on the last updated at date

Reply:
Got it, but there is no documentation available and data doesn't consistent. So I have used this logic. Can you please share any document or link to look into?